lists-service/src/main/java/au/org/ala/listsapi/controller/IngressController.java

```diff
@@ -245,7 +245,12 @@ public ResponseEntity<Object> reindex(@AuthenticationPrincipal Principal princip
             description = "Bad Request",
             content = {@Content(mediaType = "text/plain")})
       })
-  public ResponseEntity<Object> handleFileUpload(@RequestPart("file") MultipartFile file) {
+  public ResponseEntity<Object> handleFileUpload(@RequestPart("file") MultipartFile file, @AuthenticationPrincipal Principal principal) {
+    // check user logged in
+    AlaUserProfile alaUserProfile = (AlaUserProfile) principal;
+    if (alaUserProfile == null) {
```

Comment:
```suggestion
  public ResponseEntity<Object> handleFileUpload(@RequestPart("file") MultipartFile file, @AuthenticationPrincipal AlaUserProfile profile) {
    // check user logged in
    if (profile == null) {
```

Unnecessary variable? Or am I missing something?